charts/refarch-templates/README.md

```
@@ -107,6 +107,17 @@ Example:
     - name: my-pull-secret
 ```
 
+#### Refarch API Gateway
+
+The API-Gateway is included as an optional dependency inside this helm-chart. If it is not needed you can disable it by using the following example.
+> **Note:** The API-Gateway is enabled by default, due to Helm, with the standard configuration. Therefore, it is not necessary to enable it!
+
+```yaml
+refarch-gateway:
+  enable: false
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)